public/images/tinymce-icons/gridicons-chevron-down.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/><g><path fill="#2e4453" d="M20 9l-8 8-8-8 1.414-1.414L12 14.172l6.586-6.586"/></g></svg>
```

Comment:
I had to use a hex value here because coloring an svg via a css pseudo class is not possible.